src/box/box.cc

```diff
@@ -2519,6 +2531,12 @@ box_select(uint32_t space_id, uint32_t index_id,
 	uint32_t part_count = key ? mp_decode_array(&key) : 0;
 	if (key_validate(index->def, type, key, part_count))
 		return -1;
+	const char *after = pos != NULL ? pos->key : NULL;
+	if (after != NULL) {
+		uint32_t after_size = mp_decode_array(&after);
```

Comment:
We need to check that `after` is an `MP_ARRAY`.

Reply:
Stale since we decided to unpack position in `box_select_after`.